wire-ios-data-model/Resources/zmessaging.xcdatamodeld/.xccurrentversion

```diff
@@ -3,6 +3,6 @@
 <plist version="1.0">
 <dict>
 	<key>_XCCurrentVersionName</key>
-	<string>zmessaging2.109.0.xcdatamodel</string>
+	<string>zmessaging2.110.0.xcdatamodel</string>
```

Comment:
I get unit test failures because of `testThatItPerformsMigrationFrom_Between_2_24_1_and_PreLast_ToCurrentModelVersion`. Then I see it compares different `.wiredatabase` files. I've never seen this for core data, can someone please help me how to proceed here and make it work?

Reply:
1. Increase the last version of `allVersions` from `109` to `110`
2. Run the test locally (expect it to fail)
3. Follow the instructions printed in the console

Reply:
Uhh I had a mistake in the core data version (2.111.0 instead of 2.110.0) and therefore the script didn't give me the right commands 😆 thanks for the explanations!